main/checker-node.js

```diff
@@ -76,24 +76,24 @@ function maybeReportErrorToSentry (err, scopeFn) {
  * @param {Context} ctx
  */
 async function start (ctx) {
-  log.info('Starting Core...')
+  log.info('Starting Checker...')
```

Comment:
For consistency with messages printed from other places:

```suggestion
  log.info('Starting Checker Node...')
```

Reply:
a301a49